### PR TITLE
Giganews Provider Update 

### PR DESCRIPTION
**Pull request by steamGerbil**
The Giganews provider was not working for me. I have downloaded the latest ovpn files and replaced the old ones. These are now alsmost identical to the VyprVPN items, the only difference being the server endpoint

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ workflows:
 jobs:
   build-ubuntu-image:
     environment:
-      IMAGE_NAME: haugene/transmission-openvpn
+      IMAGE_NAME: steamgerbil/transmission-openvpn
```
Comment (pkishino) [inline .circleci/config.yml:56]: This is not right, please rebase and squash your commits and remove these circle config as they are pointing at the wrong image

Comment (steamGerbil) [reply .circleci/config.yml]: Sorry I thought I was on a cloned repo and not the fork, I've reverted those changes so only config changes are in. I'll rebase and push

**Comment by steamGerbil** [comment]
All changes reverted to config only and the commits have been rebased and squashed

**Comment by steamGerbil** [comment]
Saw that default.ovpn wasn't a symbolic link, corrected it

**Pull request merged**